webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts

```diff
@@ -392,7 +407,7 @@ export class ConnectionOptionsTabService extends Bootstrap {
 
     const config = contexts.getContext(connectionConfigContext);
     const stateContext = contexts.getContext(formStateContext);
-    const driver = await this.dbDriverResource.load(data.config.driverId!, ['includeProviderProperties']);
+    const driver = await this.dbDriverResource.load(data.config.driverId!, ['includeProviderProperties', 'includeMainProperties']);
```

Comment:
may be like this? i dont like when we trick TS like this with "!"

```suggestion
    if (!data.info || !data.config.driverId) {
      return;
    }

    const config = contexts.getContext(connectionConfigContext);
    const stateContext = contexts.getContext(formStateContext);
    const driver = await this.dbDriverResource.load(data.config.driverId, ['includeProviderProperties', 'includeMainProperties']);
```